src/pages/Vote/index.stories.tsx

```diff
@@ -1,14 +1,23 @@
 import { ComponentMeta } from '@storybook/react';
 import React from 'react';
+import { State } from 'react-powerplug';
```

Comment:
nit: This is done two ways in the project currently. Our own hook and this dependency.

Reply:
Yeah, we had this discussion a little while back. This package has a few additional things that can be beneficial.
I'd say it's a very minor thing and for now wouldn't spend time on it.